Add Hello Microverse on h1 tag

PeteUgwu opened the pull request:
- [ ] Set-up linters
- [ ] Ignore node_modules and test.md
- [ ] Add index.html and style.css files

PeteUgwu committed:
Add Hello Microverse on h1 tag

PeteUgwu committed:
Fix linters error

PeteUgwu committed:
Fixing linters error

PeteUgwu committed:
Merge branch 'main' into Hello

PeteUgwu committed:
Fixing

PeteUgwu committed:
Fixing

PeteUgwu committed:
Fixing

andgarzonmal reviewed (changes requested):
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️
![](https://media.giphy.com/media/3pZipqyo1sqHDfJGtz/giphy.gif)

 Hi @PeteUgwu✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them and your will be perfect.

___ 

### 💯  Highlights 💯
- Your gitHub flow is correct. ✔️
- You set up linter files and no errors. ✔️

# ♻️ Requested changes ♻️
 
- [ ] Nice work using the README file template on your project but you didn't use your information.  Kindly customize your README file with your information:

1.  Update the author section with your name and social media links 

2. Customize the get started section, as this is a simple project you can put information about how to clone the repo

3. Customize the Acknowledgments section, you can put Microverse.


- [ ] [OPTIONAL] Try adding linters on your local machine as well, this will allow you to use the [auto-correct options](https://stylelint.io/user-guide/cli#autofixing-errors)

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### Cheers and Happy coding! 💻 👏
![almost](https://media.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif)


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)

PeteUgwu committed:
Fixes on Readme file

PeteUgwu committed:
Fixing

PeteUgwu committed:
Update Readme file

PeteUgwu committed:
Update Readme file

PeteUgwu committed:
Add License file

PeteUgwu commented:
@andgarzonmal 
I have made the required changes you asked for. On the Readme file, I:
- customized the author section to bear my details
- customized the Get Started section 
- Acknowledged Microverse on the Acknowledgement section 
- Also added MIT License and cleaned up the readme file

Nyame-Wolf commented on style.css:
```diff
@@ -0,0 +1,12 @@
+  margin: 0;
+}
+
+h1 {
+  color: blue;
+  text-align: center;
+  font-size: 4rem;
+  margin-top: 3rem;
+}
```
[OPTIONAL] Kindly note that this was supposed to be a class according to the project requirements
<img width="1041" alt="Screenshot 2022-09-21 at 14 49 01" src="https://user-images.githubusercontent.com/48546992/191496658-da041c09-418b-4741-8148-3d276996c486.png">

PeteUgwu replied:
@Nyame-Wolf Thanks for the kind reminder. It is already noted

Nyame-Wolf reviewed (approved):
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @PeteUgwu  :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![](https://media.giphy.com/media/l4q7TIW8nEZYOJUf6/giphy.gif)


## Highlights :sparkles:
- Great githubflow setup
- Professional Readme
- Linters set up well and are passing

## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
